<launch>

    <arg name="start_angle" default="10" />
    <arg name="end_angle" default="50" />
    <arg name="file" default="''" />

    <node name="grasp_controller"
                pkg="proj4_pkg"
                type="grasp_controller.py"
                output="screen"
                args="-s $(arg start_angle) -e $(arg end_angle)">
    </node>

    <node name="grasp_plotter"
                pkg="proj4_pkg"
                type="grasp_plotter.py"
                output="screen"
                args="-f $(arg file)">
    </node>
</launch>
